src/test/java/jewellery/inventory/integration/ResourceInUserCrudIntegrationTest.java

```diff
@@ -285,12 +301,14 @@ void removeResourceQuantityFromUserFailsWhenInsufficientQuantity() {
     UserResponseDto createdUser = sendCreateUserRequest();
     PreciousStoneResponseDto createdResource = sendCreatePreciousStoneRequest();
     sendAddResourceInUserRequest(
-        createResourceInUserRequestDto(
-            createdUser.getId(), createdResource.getId(), RESOURCE_QUANTITY));
+        createResourcePurchaseRequestDto(
+            createdUser.getId(), createdResource.getId(), RESOURCE_QUANTITY, RESOURCE_PRICE));
 
     ResponseEntity<ResourcesInUserResponseDto> response =
         sendDeleteQuantityFromResourceInUserRequest(
-            createdUser.getId(), createdResource.getId(), RESOURCE_QUANTITY + 1);
+            createdUser.getId(),
+            createdResource.getId(),
+            RESOURCE_QUANTITY.add(BigDecimalUtil.getBigDecimal("1")));
```

Comment:
static import?

Reply:
There were quite a few more places that were not using static import.. I think I found them all and fixed them.